src/code.cloudfoundry.org/loggregator/plumbing/tls.go

```diff
@@ -21,39 +21,16 @@ func (e CASignatureError) Error() string {
 	return string(e)
 }
 
-func NewTLSConfig(
-	opts ...ConfigOption,
-) *tls.Config {
-	tlsConfig := &tls.Config{
+func NewTLSConfig() *tls.Config {
+	return &tls.Config{
 		InsecureSkipVerify: false,
```

Comment:
It is actually a min version, therefore when 1.3 comes out, we will accept that as well. We can move 1.2 to 1.0 for the min version.

I'm in favor for being explicit for `InsecureSkipVerify`.